docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+    scope:
+        restart: always
+        image: bujesse/scope:1.0
+        expose:
+            - 55850
+            - 55851
+            - 55852
+            - 55853
+        volumes:
+            - static:/app
+        entrypoint: ["poetry", "run", "hypercorn", "--bind", "0.0.0.0", "main:scope_api"]
```

Comment:
NICE TO HAVE: HTTP requests on `$SCOPE_PORT` in the `/api/` namespace should be forwarded to the hypercorn application as well.

Reply:
@MaybeJustJames Could we defer this for later or do you have a suggestion for this ?